net/zerotier/src/opnsense/service/conf/actions.d/actions_zerotier.conf

```diff
@@ -33,3 +33,39 @@ command:/usr/local/bin/zerotier-cli
 parameters: leave %s
 type:script_output
 message:Leaving Zerotier Network
+
+[info]
+command:/usr/local/bin/zerotier-cli info
+parameters:
+type:script_output
+message:Listing Zerotier Info
+
+[listnetworks]
+command:/usr/local/bin/zerotier-cli listnetworks
+parameters:
+type:script_output
+message:Listing Zerotier Networks
+
+[listpeers]
+command:/usr/local/bin/zerotier-cli listpeers
+parameters:
+type:script_output
+message:Listing Zerotier Peers
+
+[info_json]
```

Comment:
if you use "info.json" you can type/use "info json" in the code, maybe that makes things a tiny bit easier, especially when non-json variants of the commands exist